compute/api/Compute.Samples/GetUsageExportBucketAsync.cs

```diff
@@ -37,8 +39,8 @@ public async Task<Project> GetUsageExportBucketAsync(string projectId = "your-pr
         else if (string.IsNullOrEmpty(project.UsageExportLocation.ReportNamePrefix))
         {
             Console.WriteLine(
-                $"A null or empty value for {nameof(UsageExportLocation.ReportNamePrefix)} " +
-                $"will result in reports being prefixed with `usage_gce`.");
+                $"Setting {nameof(UsageExportLocation.ReportNamePrefix)} " +
+                $"to empty value causes the report to have the default prefix of `usage_gce`.");
```

Comment:
null and empty are two different values in C#, they have the same effect on the prefix. Here and everywhere.
```suggestion
                $"to null or empty values causes the report to have the default prefix of `usage_gce`.");
```

Reply:
Possibly "to null or an empty string" - but generally, yes.